doc/set-of-runs.mzQC.md

```diff
@@ -0,0 +1,312 @@
+For the purpose of this example, we will just use two `setQuality` objects (there could be none, only one or more than two though). How you define (and name) each set, is up to you and depends on your experimental design.
+A `setQuality` represents QC data that must be viewed in the context of all the runs of this set/group. I.e. the data is only valid within the context of the runs it comprises. E.g. it would be invalid to define a set of three runs and report their individual MS1 scan counts as a 3-tuple -- because this information can clearly be attributed to individual runs.
+Similar to `runQuality`, a `setQuality` also contains `metadata` about the set of runs (its input file**s**, the software used). 
+You can give the set a unique name using the `label` attribute. Here is how a `setQuality` object looks like:
+```
+      {
+        "metadata": {
+          "label": "TechRepl:1,2,3_healthy"
+          "inputFiles": 
+            ...
+        },
+        "qualityMetrics": [
+            ...
+        ]
+      }
+```
+The `inputFiles` consist of an array of `inputFile` objects, describing the source files with structured information about the file's name, format, location and other properties, defined via cv terms. 
+```
+          "inputFiles": [
+            {
+              "name": "techRep1_healthy",
+              "location": "c:\msdata\techRep1_healthy.raw",
+              ...
+            },
+            {
+              "name": "techRep2_healthy",
+              "location": "c:\msdata\techRep2_healthy.raw",
+              ...
+            },
+            {
+              "name": "techRep3_healthy",
+              "location": "c:\msdata\techRep3_healthy.raw",
+              ...
+            }
+          ]
+```
+The `inputFile` object is only sketched here. It can contain a lot more information, such as file format and further properties. See the full example below or `individual-runs.mzQC` for details.
+
+In  `qualityMetrics`, we will store the actual QC information for a particular `setQuality`. Each `qualityMetric` has an `accession` and the corresponding `name` are defined by the QC CV (see `qc-cv.obo`) and should be represented exactly as stated in the .obo file. The `value` carries the actual information.
+Metric `value`s can be either single values, tuple of values, or matrices or tables (shown in other examples). 
+
+Here we show two metrics: the first describes the relative total intensity of all contaminant proteins (from all runs in the set) -- the higher the value the more contaminants are present in the samples of the set.
+```
+            "accession": "QC:0000000",
+            "name": "Protein contaminant intensity ratio",
+            "value": 0.25
+```
+... and the second metric is a single column of a PCA result matrix. This metric is a bit special because it only partially describes the data.
```

Comment:
I find this a weird way to encode PCA values. Wouldn't a table be a more natural representation? That way all PCA values are stored together.

Reply:
I totally agree!
But there is no way to encode the groups in a natural way that I can think of. There is another solution, but its also a bit complicated:
use a setQuality which contains all Raw files. In there, store the matrix as you suggest, and then use another qualityMetric to define the subgroups and their names.

Reply:
Sorry I missed the call this morning. Stupid mistake. 🤦‍♂️ So I'm not sure to what extent this has been discussed, but at first thought I would prefer your second solution over the current one.